scripts/start/readCheckScenarioConfig.R

```diff
@@ -27,18 +27,27 @@ readCheckScenarioConfig <- function(filename, remindPath = ".", testmode = FALSE
             paste0(rownames(scenConf)[regionname], collapse = ", "),
             " – Titles with three capital letters or 'glob' may be confused with region names by magclass. Stopping now.")
   }
-  containsdot <- grep("\\.", rownames(scenConf))
+  containsdot <- grepl("\\.", rownames(scenConf))
   if (any(containsdot)) {
     warning("These titles contain a dot: ",
             paste0(rownames(scenConf)[containsdot], collapse = ", "),
             " – GAMS would not tolerate this, and quit working at a point where you least expect it. Stopping now.")
   }
+  whitespaceerrors <- 0
+  for (path_gdx in intersect(names(path_gdx_list), names(scenConf))) {
+    haswhitespace <- grepl("\\s", scenConf[[path_gdx]])
+    if (any(haswhitespace)) {
+      warning("The ", path_gdx, " cells of these runs contain whitespaces: ", paste0(rownames(scenConf)[haswhitespace], collapse = ", "),
+              " – scripts will fail to find corresponding runs and gdx files. Stopping now.")
+      whitespaceerrors <- whitespaceerrors + sum(haswhitespace)
+    }
+  }
+  errorsfound <- sum(toolong) + sum(regionname) + sum(containsdot) + whitespaceerrors
```

Comment:
No half measures.
```
path_name_errors <- 0
for (path_gdx in intersect(names(path_gdx_list), names(scenConf))) {
  has_illeagal_chars <- grepl("[^[:alnum:]_\\.-]", scenConf[[path_gdx]])
  if (any(has_illeagal_chars)) {
    warning("The ", path_gdx, " cells of these runs contain illeagal characters: ",
            paste0(rownames(scenConf)[has_illeagal_chars], collapse = ", "),
            " – scripts will fail to find corresponding runs and gdx files. Stopping now.")
    path_name_errors <- path_name_errors + sum(has_illeagal_chars)
  }
}
errorsfound <- sum(toolong) + sum(regionname) + sum(containsdot) + path_name_errors
```
Allow characters, numbers, "-", "_", and ".".  I have never seen anything else being used.

Reply:
I mean, at least `/` should be allowed, and for Windows users `\` or so, I guess :)

Reply:
And I would rather treat whitespaces independently, because they are quite difficult to recognize, at while editing csv files with Excel/OpenOffice. And we should at least tell the users which characters are legal :)

Reply:
And I wonder whether it is necessary. If the file `@asdf.gdx` exists and the scripts finds it, what is the problem? If it doesn't find it, it will fail anyway.

Reply:
> I mean, at least / should be allowed, and for Windows users \ or so, I guess :)

True.  I was thinking of scenario names.